src/modules/mavlink/module.yaml

```diff
@@ -142,14 +150,29 @@ parameters:
             description:
                 short: Broadcast heartbeats on local network for MAVLink instance ${i}
                 long: |
-                    This allows a ground control station to automatically find the drone 
+                    This allows a ground control station to automatically find the drone
                     on the local network.
 
             type: enum
             values:
                 0: Never broadcast
                 1: Always broadcast
-                2: Only multicast        
+                2: Only multicast
             num_instances: *max_num_config_instances
             default: [1, 0, 0]
             requires_ethernet: true
+
+        MAV_${i}_FLOW_CONT:
+            description:
+                short: Enable serial flow control for instance ${i}
+                long: |
+                    If 0, flow control forced off. If 1, flow control forced on.
```

Comment:
The long description should probably be explaining the higher level context (if any), and not the specific values.